### ISSUE TITLE
Data matrix support?

### ISSUE THREAD

**Issue by crazyimp** [mchehab/zbar#81]
Might be possible to use https://github.com/dmtx/libdmtx to add data matrix support?

**Comment by jasp00** [comment]
Yes, it might be possible.

**Comment by mchehab** [comment]
> Might be possible to use https://github.com/dmtx/libdmtx to add data matrix support?

Yes, but someone has to work on the patches adding support for it.

**Comment by oussama-he** [comment]
Any plans to add the support for data matrix barcodes type?
